packages/ember-metal/tests/utils_test.js

```diff
@@ -10,6 +10,7 @@ QUnit.module('Ember Metal Utils');
 QUnit.test('inspect outputs the toString() representation of Symbols', function() {
   // Symbol is not defined on pre-ES2015 runtimes, so this let's us safely test
   // for it's existence (where a simple `if (Symbol)` would ReferenceError)
+  /* jshint latedef: false */
```

Comment:
Can you remove this and rebase now that https://github.com/emberjs/ember.js/pull/14531 is landed?